p11kit/attribute.go

```diff
@@ -135,7 +135,36 @@ func signECDSA(priv crypto.Signer, m mechanism, data []byte) ([]byte, error) {
 	if !m.noParams() {
 		return nil, fmt.Errorf("CKM_ECDSA does not take any parameters: %w", errArgumentsBad)
 	}
-	return priv.Sign(rand.Reader, data, crypto.Hash(0))
+	sig, err := priv.Sign(rand.Reader, data, crypto.Hash(0))
+	if err != nil {
+		return nil, err
+	}
+
+	// PKCS11 use raw R, S value as signature, priv.Sign return a DER encoded R, S sequence
+	// convert DER sequence to raw R, S
+	// http://docs.oasis-open.org/pkcs11/pkcs11-curr/v2.40/cs01/pkcs11-curr-v2.40-cs01.html#_Toc399398879
+	var sigDecode []*big.Int
+	_, err = asn1.Unmarshal(sig, &sigDecode)
```

Comment:
nit: this can use a struct

```
type sigDecode struct {
    R *big.Int
    S *big.Int
}
if _, err := asn1.Unmarshal(sig, &sigDecode); err != nil {
    // ...
}
```